test/unit/mongo/MongoTripodViewsTest.php

```diff
@@ -342,13 +379,39 @@ public function testGetViewForResourcesInvokesViewGenerationForMissingResources(
         $context = "http://someContext";
 
         $mockDb = $this->getMock("MongoDB", array("selectCollection"),array(new MongoClient(),"test"));
-        $mockColl = $this->getMock("MongoCollection", array("findOne"),array($mockDb,$this->tripod->collection->getName()));
+        $mockColl = $this->getMock("MongoCollection", array("findOne"),array($mockDb,$this->tripod->getPodName()));
+        $mockViewColl = $this->getMock("MongoCollection", array("findOne"),array($mockDb,VIEWS_COLLECTION));
 
-        $mockDb->expects($this->once())->method("selectCollection")->will($this->returnValue($mockColl));
+        $mockDb->expects($this->any())->method("selectCollection")->will($this->returnValue($mockColl));
```

Comment:
Why?  The purpose of this test isn't to ensure that selectCollection is called, and, in fact, it probably won't be.

However, we can't create a MongoCollection without a MongoDB, so we want to be sure that MongoDB doesn't do anything with the database.